zowex/Cargo.toml

```diff
@@ -1,6 +1,6 @@
 [package]
 name = "zowe"
-version = "1.1.0"
```

Comment:
Should we bump to version 1.1.1 or 1.2.0? I'm kind of on the fence - it is an enhancement but only for Mac. Not sure whether a minor bump is warranted since nothing has changed for Windows and Linux users?

Reply:
I think it makes sense to update the version to 1.2.0 😋